packages/sampler/stories/default/VirtualGridList.js

```diff
@@ -17,7 +18,8 @@ const
 		'&quot;noAnimation&quot;': 'noAnimation'
 	},
 	prop = {
-		direction: {'horizontal': 'horizontal', 'vertical': 'vertical'}
+		direction: {'horizontal': 'horizontal', 'vertical': 'vertical'},
```

Comment:
`{horizontal: 'horizontal', vertical: 'vertical'}` would be better.

Reply:
Done.